src/components/PluginDependencies.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import {Link} from 'gatsby';
 import {Modal, ModalHeader, ModalBody} from 'reactstrap';
 
-function PluginDependencies({dependencies} ) {
+function PluginDependencies({dependencies, reverseDependencies} ) {
```

Comment:
reverse dependencies are things that depend on it?

Reply:
Yes, I borrrowed the term from [Ruby Gems](https://rubygems.org/gems/parallel/reverse_dependencies) but in the UI it's shown as "dependent plugins" because that may make more sense to the users.